README.md

```diff
@@ -241,22 +260,31 @@ You can PATCH services to add or update a service, the following json body is ex
 
 Example using HTTPie:
 
-    http PATCH :5000/services\?env=stage service=my-service url=http://example.com/my-service.js
+```sh
+http PATCH :5000/services\?env=stage service=my-service url=http://example.com/my-service.js
+```
 
 Example using cURL:
 
-    curl -d '{ "service":"my-service","url":"http://example.com/my-service.js" }' -X PATCH localhost:5000/services\?env=beta -H "Accept: application/json" -H "Content-Type: application/json"
+```sh
+curl -d '{ "service":"my-service","url":"http://example.com/my-service.js" }' -X PATCH localhost:5000/services\?env=beta -H "Accept: application/json" -H "Content-Type: application/json"
+```
 
 #### DELETE /services/{SERVICE_NAME}?env=alpha
 
 You can remove a service by sending a DELETE with the service name. No request body needs to be sent. Example:
 
-    DELETE /services/my-service
+```sh
+DELETE /services/my-service
+```
 
 Example using HTTPie:
 
-    http DELETE :5000/services/my-service
+```sh
+http DELETE :5000/services/my-service
+```
 
 Example using cURL:
 
-    curl -X DELETE localhost:5000/services/my-service
+```sh
+curl -X DELETE localhost:5000/services/my-service
```

Comment:
nit but I think you need to terminate the triple tickmarks here

Reply:
and i thought that i was the typo nazi.